Dockerfile

```diff
@@ -26,58 +26,49 @@ ARG RVM_VERSION=stable
 ENV RVM_VERSION=${RVM_VERSION}
 
 # RMV user to create
+# Optional: child images can change to this user, or add 'rvm' group to other user
 ARG RVM_USER=rvm
 ENV RVM_USER=${RVM_USER}
 
-# Install RVM
-RUN apt update \
-    && apt install -y \
-       curl \
-       git \
-       gnupg2 \
+# Install RVM dependencies
+RUN sed -i 's~http://archive\(\.ubuntu\.com\)/ubuntu/~mirror://mirrors\1/mirrors.txt~g' /etc/apt/sources.list \
+    && apt-get update -qq \
+    && apt-get install -qy --no-install-recommends \
+       ca-certificates curl netbase \
+       git openssh-client \
+       gnupg dirmngr \
```

Comment:
It appears that in Ubuntu 18 and later, both `gnupg2` and `gnupg` packages are both version 2.2.4. Can you elaborate on why you've chosen the `gnupg` package, which appears to be "secretly" `gnupg2`, rather than explicitly the version 2 package?

Reply:
Because gnupg is smaller than gnupg2, and they are both able to verify the RVM installer, so I choice gnupg.

The official base-image is referred to use `gnupg` like:
https://github.com/docker-library/buildpack-deps/blob/master/bionic/curl/Dockerfile

Reply:
Ok, that's fine. They do appear to be the same version, so it's just a convention.